client/src/utils/get-words.js

```diff
@@ -1,6 +1,14 @@
-const config = require('../../../config/env.json');
+import env from '../../../config/env.json';
 
-const words = require(`../../i18n/locales/${config.clientLocale}/motivation.json`);
+import english from '../../i18n/locales/english/motivation.json';
+import espanol from '../../i18n/locales/espanol/motivation.json';
+import chinese from '../../i18n/locales/chinese/motivation.json';
+
+// The idea here is to use static imports to allow for tree-shaking and dead
```

Comment:
Is this part of this pr? it seems like i18n related.

Reply:
Ahem.  I'll remove it.